mmv1/templates/terraform/constants/network_services_gateway.go.erb

```diff
@@ -0,0 +1,109 @@
+<%# The license inside this block applies to this file
+	# Copyright 2023 Google Inc.
+	# Licensed under the Apache License, Version 2.0 (the "License");
+	# you may not use this file except in compliance with the License.
+	# You may obtain a copy of the License at
+	#
+	#     http://www.apache.org/licenses/LICENSE-2.0
+	#
+	# Unless required by applicable law or agreed to in writing, software
+	# distributed under the License is distributed on an "AS IS" BASIS,
+	# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+	# See the License for the specific language governing permissions and
+	# limitations under the License.
+-%>
+// Checks if there is another gateway under the same location.
+func gatewaysSameLocation(d *schema.ResourceData, config *transport_tpg.Config, billingProject, userAgent string) ([]interface{}, error) {
+	log.Print("[DEBUG] Looking for gateways under the same location.")
+    var gateways []interface{}
```

Comment:
nit: spacing: should be tabs instead of spaces

Reply:
Done